Commands/Commands/queue.js

```diff
@@ -270,6 +270,49 @@ commands.registerVote = {
         return
       }
       switch (doc.type) {
+      case 'upvoteOnly' : {
+        if (reaction.id === '302138464986595339') {
+          getMail(uv, user.id).then(f => {
+            uv.v1.loginAs(f).then(c => {
+              c.post(`forums/${config.uservoice.forumId}/suggestions/${doc.UvId}/votes.json`, {
+                to: 1
+              }).then((s) => {
+                if (user !== null) {
+                  genlog.log(bot, user, {
+                    message: 'Top10-voted',
+                    affected: doc.UvId
+                  })
+                }
+              }).catch(e => {
+                if (e.statusCode === 404) {
+                  logger.log(bot, {
+                    cause: 'top10_vote',
+                    message: (e.message !== undefined) ? e.message : JSON.stringify(e)
+                  }, e)
```

Comment:
What's the point in sending the error string to #bot-error? According this we already know its a missing suggestion.